lib/chef/dist.rb

```diff
@@ -0,0 +1,18 @@
+class Chef
+  class Dist
+    # When referencing a product directly, like Chef (Now Chef Infra)
+    PRODUCT = "Chef Infra".freeze
+
+    # The client's alias (chef-client)
+    CLIENT = "chef-client".freeze
+
+    # Name used for certain directories. Merge with chef_executable?
+    GENERIC = "chef".freeze
```

Comment:
The only place this is being used it should probably be Chef::PRODUCT instead and Chef::GENERIC should be dropped (not wild about having a "GENERIC" in any case since that "smells" like something that will eventually need to be broken up into multiple different concerns)

Reply:
I think all comments at this time have been addressed with the latest commit